pkg/solana/config_tracker.go

```diff
@@ -27,25 +28,37 @@ func configFromState(state State) types.ContractConfig {
 		accounts = append(accounts, types.Account(o.Transmitter.String()))
 	}
 
+	// calculate OnchainConfig (currently not calculated onchain, but required for libocr)
```

Comment:
Let's please add more docs explaining this comment. 

- Why is that?
- What does "currently" mean?
- Do we expect it to change?